website/pages/docs/configuration/storage/raft.mdx

```diff
@@ -52,4 +52,38 @@ set `ha_enabled = true`.
 - `node_id` `(string: "")` - The identifier for the node in the Raft cluster.
   This value can be overridden by setting the `VAULT_RAFT_NODE_ID` environment variable.
 
+- `performance_multiplier` `(integer: 0)` - An integer multiplier used by
+  servers to scale key Raft timing parameters. Omitting this value or setting it
+  to 0 uses default timing described below. Lower values are used to tighten
+  timing and increase sensitivity while higher values relax timings and reduce
+  sensitivity. Tuning this, affects the time it takes Vault to detect leader
```

Comment:
I'm being grammar-picky, but you don't need a comma ("Tuning this affects"). I would also put the "Tuning this" sentence second, maybe? To me the timing and sensitivity concerns didn't make any sense until I'd read the "Tuning this" sentence.

> An integer multiplier used by servers to scale key Raft timing parameters. Tuning this affects the time it takes Vault to detect leader...Omitting this value or setting it to 0 uses default timing described below. Lower values are used to tighten timing and increase sensitivity while higher values relax timings and reduce sensitivity.